backend/test/game/dice_handler_test.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from backend.src.game.dice_handler import DiceHandler
+import backend.src.game.dice as dice
+
+
+@pytest.fixture(autouse=True)
+def dice_handler():
+    dice_handler = DiceHandler()
+    return dice_handler
+
+
+def test_start_turn(dice_handler):
+    for die in dice_handler.dice_values:
+        assert die in dice.DieValue
+
+
+def test_re_roll_one_die(dice_handler):
+    dice_handler.start_turn()
+    start_value = dice_handler.dice_values[0]
+    roll_changed_value = False
+    # since technically possible re-roll returns same roll, run 10 times and quit on successful result
+    for _ in range(10):
```

Comment:
Please don't rely on random within tests.
Mock out the roll() to force the die to roll to a specific value you can assert it has changed to.
1. Mock out rollMany() to return a set, like 1,2,3,atk,heal,energy
2. Mock out roll() to return something fixed, like energy
3. Call re_roll_dice(0) 
4. assert energy,2,3,atk,heal,energy

Reply:
Thanks for the steps!